__tests__/index.test.js

```diff
@@ -119,7 +119,7 @@ describe("findById()", () => {
   });
 });
 
-describe("filterByGenre()", () => {
+describe.only("filterByGenre()", () => {
```

Comment:
please remember to take ".only" off your tests before you submit in the future. It makes it a lot easier for us to grade and see what the tests are doing when they're all able to run :)
```suggestion
describe("filterByGenre()", () => {
```